### ISSUE TITLE
dictionary problem with coding of greek characters

### ISSUE THREAD

**Issue by arnoulfos** [koreader/koreader#7573]
* KOReader version: 2021.03
* Device: android 7

#### Issue
Can someone explain me why ἀθυμία and ἀθυμία, give a different search result in the koreader dictionary while it is the same word, it may be a problem of coding of characters but I don t understand which
##### `crash.log` (if applicable)
`crash.log` is a file that is automatically created when KOReader crashes. It can
normally be found in the KOReader directory:

* `/mnt/private/koreader` for Cervantes
* `koreader/` directory for Kindle
* `.adds/koreader/` directory for Kobo
* `applications/koreader/` directory for Pocketbook

Android won't have a crash.log file because Google restricts what apps can log, so you'll need to obtain logs using `adb logcat KOReader:I ActivityManager:* AndroidRuntime:* DEBUG:* *:F`.

Please try to include the relevant sections in your issue description.
You can upload the whole `crash.log` file on GitHub by dragging and
dropping it onto this textbox.

**Comment by arnoulfos** [comment]
I  got the explanation in the first the ι is accented with an ancient greek accent called ὀξεία, in the second with a modern greek accent called τόνος. although they look similar by looking closely the τόνος is more vertical. I would be very useful if the koreader dictionary could treat those 2 accents as the same.Thanks in advance.

**Comment by Markismus** [comment]
`sdcv` the CLI Stardict dictionary doesn't know any accents. Rather it knows UTF-8. And in UTF-8 those differently accented iota symbols are different codepoints. 

Enabling fuzzy search in the dictionary settings menu might help you. If I disabled it at the command line, I got zero hits with the currently installed dictionary, while enabled one of the words resulted in a proper subset of the other:
```
$ sdcv ἀθυμία -e
Nothing similar to ἀθυμία, sorry :(
$ sdcv ἀθυμία -e
Nothing similar to ἀθυμία, sorry :(
$
$ sdcv ἀθυμία 
Found 2 items, similar to ἀθυμία.
0)Πάπυρος (El-El)-->αθυμία
1)Πάπυρος (El-El)-->θυμία
Your choice[-1 to abort]: ^C
$ sdcv ἀθυμία
Found 9 items, similar to ἀθυμία.
0)Πάπυρος (El-El)-->αθυμία
1)Πάπυρος (El-El)-->θυμία
2)Πάπυρος (El-El)-->θυία
3)Πάπυρος (El-El)-->θυεία
4)Πάπυρος (El-El)-->θυμίαμα
5)Πάπυρος (El-El)-->θυμίαση
6)Πάπυρος (El-El)-->θυμίασμα
7)Πάπυρος (El-El)-->θυσία
8)Πάπυρος (El-El)-->φυμία
Your choice[-1 to abort]: ^C
[mark@debiel koreader-appimage]$ 
```

**Comment by arnoulfos** [comment]
Yes thanks, the fuzzy search helps, but in many case is returning too much entries many of them irrelevant, that s why it would be extremely useful an option for accent insensitive search, without fuzzy search

**Comment by Markismus** [comment]
You could prepare an accentless dictionary. I think [Rosen tried something along that line](http://www.mobileread.mobi/forums/showthread.php?t=256360): He created a dictionary that would return an accentless version of a word, which could than be searched. It would take two lookups, but that's just two clicks in Koreader.

**Comment by Commodore64user** [comment]
duplicate of #7540
